Sources/UInt128/UInt128.swift

```diff
@@ -663,6 +663,9 @@ extension UInt128 : ExpressibleByStringLiteral {
     internal static func _valueFromString(_ value: String) -> UInt128? {
         let radix = UInt128._determineRadixFromString(value)
         let inputString = radix == 10 ? value : String(value.dropFirst(2))
+        guard inputString.isEmpty == false else {
+            return nil
+        }
```

Comment:
What was happening when an empty string was being passed in? Did we not have a test covering this? If we did, was it failing without this change? If we didn't, could you explain the scenario where this is an issue and also introduce a test to make sure we never introduce a regression on this?

Reply:
@gongzhang; mind looking at this question again, I'm not sure if you've spoken to it yet?